src/sentry/models/project.py

```diff
@@ -415,5 +415,16 @@ def copy_settings_from(self, project_id):
             return False
         return True
 
+    @staticmethod
+    def valid_platform(value):
```

Comment:
```suggestion
    def is_valid_platform(value):
```

I think it’s a bit more clearly named this way to indicate it’s asking a question.